src/store/ping.ts

```diff
@@ -1,8 +1,9 @@
 import { writable } from 'svelte/store';
-import { PingStats, RoundtripTimer } from '@lib/ping/PingStats';
+import { PingStats,  RoundTripTimer } from '@lib/ping/PingStats';
 
+// can a store take a dependency?
```

Comment:
> "// can a store take a dependency?"

Probably, what are you thinking?

Reply:
ping stats contructor takes one, so was wondering if that can be added?